extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -80,13 +98,26 @@ function BaseCarouselWithRef(
 
   const disableForDir = (dir) =>
     !loop && (currentSlide + dir < 0 || currentSlide + dir >= length);
+
+  const [hadTouch, setHadTouch] = useState(false);
+  const hideControls = useMemo(() => {
+    if (controls === Controls.NEVER) {
+      return true;
+    }
+    if (controls === Controls.ALWAYS) {
+      return false;
+    }
+    return hadTouch;
+  }, [hadTouch, controls]);
+
   return (
     <ContainWrapper size={true} layout={true} paint={true} {...rest}>
       <Scroller
         loop={loop}
         restingIndex={currentSlide}
         setRestingIndex={setRestingIndex}
         ref={scrollRef}
+        onTouchStart={() => setHadTouch(true)}
```

Comment:
So, is this mobile only feature? Or should we use `onPointerDown` instead?

Reply:
Mobile only - 0.1 looks for `touchstart` also